<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Tools and techniques for single-cell RNA-seq data</title>
  <meta name="description" content="Tools and techniques for single-cell RNA-seq data">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Tools and techniques for single-cell RNA-seq data" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Tools and techniques for single-cell RNA-seq data" />
  
  
  

<meta name="author" content="Luke Zappia">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="index.html">
<link rel="next" href="2-the-scrna-seq-tools-landscape.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Front matter</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#declaration"><i class="fa fa-check"></i>Declaration</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#preface"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#rna-sequencing"><i class="fa fa-check"></i><b>1.1</b> RNA sequencing</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-introduction.html"><a href="1-introduction.html#capture-and-reverse-transcription"><i class="fa fa-check"></i><b>1.1.1</b> Capture and reverse transcription</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-introduction.html"><a href="1-introduction.html#high-throughput-sequencing"><i class="fa fa-check"></i><b>1.1.2</b> High-throughput sequencing</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-introduction.html"><a href="1-introduction.html#analysis-of-rna-seq-data"><i class="fa fa-check"></i><b>1.1.3</b> Analysis of RNA-seq data</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#single-cell-capture-technologies"><i class="fa fa-check"></i><b>1.2</b> Single-cell capture technologies</a><ul>
<li class="chapter" data-level="1.2.1" data-path="1-introduction.html"><a href="1-introduction.html#droplet-based-cell-capture"><i class="fa fa-check"></i><b>1.2.1</b> Droplet based cell capture</a></li>
<li class="chapter" data-level="1.2.2" data-path="1-introduction.html"><a href="1-introduction.html#unique-molecular-identifiers"><i class="fa fa-check"></i><b>1.2.2</b> Unique Molecular Identifiers</a></li>
<li class="chapter" data-level="1.2.3" data-path="1-introduction.html"><a href="1-introduction.html#recent-advances"><i class="fa fa-check"></i><b>1.2.3</b> Recent advances</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#features-of-single-cell-rna-seq-data"><i class="fa fa-check"></i><b>1.3</b> Features of single-cell RNA-seq data</a></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#analysing-scrna-seq-data"><i class="fa fa-check"></i><b>1.4</b> Analysing scRNA-seq data</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-introduction.html"><a href="1-introduction.html#pre-processing-and-quality-control"><i class="fa fa-check"></i><b>1.4.1</b> Pre-processing and quality control</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-introduction.html"><a href="1-introduction.html#integrating-multiple-datasets"><i class="fa fa-check"></i><b>1.4.2</b> Integrating multiple datasets</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-introduction.html"><a href="1-introduction.html#grouping-cells"><i class="fa fa-check"></i><b>1.4.3</b> Grouping cells</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-introduction.html"><a href="1-introduction.html#ordering-cells"><i class="fa fa-check"></i><b>1.4.4</b> Ordering cells</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-introduction.html"><a href="1-introduction.html#gene-detection-and-interpretation"><i class="fa fa-check"></i><b>1.4.5</b> Gene detection and interpretation</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-introduction.html"><a href="1-introduction.html#kidney-development"><i class="fa fa-check"></i><b>1.5</b> Kidney development</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-introduction.html"><a href="1-introduction.html#structure-and-function"><i class="fa fa-check"></i><b>1.5.1</b> Structure and function</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-introduction.html"><a href="1-introduction.html#stages-of-development"><i class="fa fa-check"></i><b>1.5.2</b> Stages of development</a></li>
<li class="chapter" data-level="1.5.3" data-path="1-introduction.html"><a href="1-introduction.html#growing-kidney-organoids"><i class="fa fa-check"></i><b>1.5.3</b> Growing kidney organoids</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-the-scrna-seq-tools-landscape.html"><a href="2-the-scrna-seq-tools-landscape.html"><i class="fa fa-check"></i><b>2</b> The scRNA-seq tools landscape</a></li>
<li class="chapter" data-level="3" data-path="3-simulating-scrna-seq-data.html"><a href="3-simulating-scrna-seq-data.html"><i class="fa fa-check"></i><b>3</b> Simulating scRNA-seq data</a><ul>
<li class="chapter" data-level="3.1" data-path="3-simulating-scrna-seq-data.html"><a href="3-simulating-scrna-seq-data.html#introduction-1"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-simulating-scrna-seq-data.html"><a href="3-simulating-scrna-seq-data.html#splatter-publication"><i class="fa fa-check"></i><b>3.2</b> Splatter publication</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-visualising-clustering-across-resolutions.html"><a href="4-visualising-clustering-across-resolutions.html"><i class="fa fa-check"></i><b>4</b> Visualising clustering across resolutions</a></li>
<li class="chapter" data-level="5" data-path="5-analysis-of-kidney-organoid-scrna-seq-data.html"><a href="5-analysis-of-kidney-organoid-scrna-seq-data.html"><i class="fa fa-check"></i><b>5</b> Analysis of kidney organoid scRNA-seq data</a></li>
<li class="chapter" data-level="6" data-path="6-conclusion.html"><a href="6-conclusion.html"><i class="fa fa-check"></i><b>6</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Tools and techniques for single-cell RNA-seq data</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="introduction" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<div id="rna-sequencing" class="section level2">
<h2><span class="header-section-number">1.1</span> RNA sequencing</h2>
<ul>
<li>Central dogma</li>
<li>Why RNA-seq?</li>
</ul>
<div id="capture-and-reverse-transcription" class="section level3">
<h3><span class="header-section-number">1.1.1</span> Capture and reverse transcription</h3>
<ul>
<li>PolyA capture</li>
<li>Ribosomal depletion</li>
</ul>
</div>
<div id="high-throughput-sequencing" class="section level3">
<h3><span class="header-section-number">1.1.2</span> High-throughput sequencing</h3>
<ul>
<li>Illumina sequencing
<ul>
<li>Sequence by synthesis</li>
</ul></li>
</ul>
</div>
<div id="analysis-of-rna-seq-data" class="section level3">
<h3><span class="header-section-number">1.1.3</span> Analysis of RNA-seq data</h3>
<ul>
<li>Experimental design</li>
<li>Negative binomial</li>
<li>Normalisation</li>
<li>Differential expression testing</li>
</ul>
</div>
</div>
<div id="single-cell-capture-technologies" class="section level2">
<h2><span class="header-section-number">1.2</span> Single-cell capture technologies</h2>
<ul>
<li>First protocol</li>
<li>Fluidigm</li>
</ul>
<p>The first scRNA-seq protocol was published in 2009, just a year after the first
bulk RNA-seq publication. While this approach allowed measurements of the
transcriptome in individual cells it required manual manipulation and was
restricted to inspecting a few precious cells. Further studies quickly showed
that cell types could be identified without sorting cells and approaches were
developed to allow unbiased capture of the whole transcriptome. Since then many
scRNA-seq protocols have been developed including …. The first commercially
available cell capture platform was the Fluidigm C1. This system uses
microfluidics to passively separate cells into individual wells on a plate
where they are lysed, reverse-transcribed and the collected cDNA is PCR
amplified. After this stage the product is extracted from the plate and
libraries prepared for Illumina sequencing. Most C1 data has been produced
using a 96 well plate but more recently an 800 well plate has become available,
greatly increasing the number of cells that can be captured at a time. One of
the disadvantages of microfluidic cell capture technologies is that the chips
have a fixed size window, meaning that only cells of a particular sizes can be
captured in a single run. However, as cells are captured in individual wells
they can be imaged before lysis, potentially identifying damaged or broken
cells, empty wells or wells containing more than one cell. Capturing multiple
cells is a known issue, with Macosko et al. finding that when preparing a
mixture of mouse and human cells 30 percent of the resulting libraries
contained transcripts from both species but only about a third of these
doublets were visible in microscopy images[Macosko2015-rl]. The newer Polaris
system from Fluidigm also uses microfluidics to capture cells but can select
particular cells based on staining or fluorescent reporter expression and then
hold them for up to 24 hours while introducing various stimuli. The cells can be
imaged during this time before being lysed and prepared for RNA sequencing. This
platform provides opportunities for a range of experiments that aren’t possible
using other capture technologies.</p>
<div id="droplet-based-cell-capture" class="section level3">
<h3><span class="header-section-number">1.2.1</span> Droplet based cell capture</h3>
<ul>
<li>Drop-seq</li>
<li>Indrop</li>
<li>10x Chromium</li>
</ul>
<p>An alternative to using microfludics to capture cells in wells is to capture
them in nano-droplets. A dissociated cell mixture is fed into a microfluidic
device while at another input beads coated in primers enter. The device is
designed to form aqueous droplets within mineral and the inputs are arranged so
that cells and beads can be simultaneously captured within a droplet. When this
happens the reagents carried along with the bead lyse the cell and any PolyA
tagged RNA molecules present can bind to the capture probes on the bead. Reverse
transcription and PCR amplification then begins and an individual cDNA library
is produced for each cell, tagged with the unique barcode sequence present on
the bead. The main advantage of droplet-based capture technologies is the
ability to capture many more cells at one time, up to tens of thousands. These
approaches are also less selective about cell size and produce less doublets. As
a result they are much cheaper per a cell, although as sequencing costs are
fixed studies using droplet-based captures typically sequence individual cells
at a much lower depth.</p>
<p>Droplet-based capture was popularised by the publication of the Drop-seq and
InDrop platforms in 2015. This are both DIY systems and although they differ in
how the beads are produced, when the droplets are broken and some aspects of
the chemistry they can both be constructed on a lab bench from syringes,
automatic plungers, a micro scope and a small custom-made microfluidic chip. A
similar commercially available platform is the 10x Genomics Chromium device
which automates and streamlines much of the process. This device uses
droplet-based technologies for a range of applications including capture of
cells for scRNA-seq. More specialised captures, such as those aimed at
profiling immune cell receptors are also possible and the company has recently
announced kits for single-cell ATAC-seq capture.</p>
</div>
<div id="unique-molecular-identifiers" class="section level3">
<h3><span class="header-section-number">1.2.2</span> Unique Molecular Identifiers</h3>
<ul>
<li>Why?</li>
<li>How they work</li>
</ul>
<p>In contrast to plate-based capture methods, which often provide reads along the
length of transcripts, droplet-based capture methods typically employ protocols
which include short random nucleotide sequences known as Unique Molecular
Identifiers (UMIs). Individual cells contain very small amounts of RNA and to
obtain enough cDNA a PCR amplification step is necessary. Depending on their
nucleotide sequence different transcripts may be amplified at different rates
which can distort their relative proportions within a library. UMIs attempt to
improve the quantification of gene expression by allowing the removal of PCR
duplicates produced during amplification. The nucleotide probes used in
droplet-based capture protocols include a PolyT sequence which binds to mature
mRNA molecules, a barcode sequence which is the same for every probe on a bead
and 8-10 bases of UMI sequence which is unique to each probe. The UMI sequences
are long enough that the probability of capturing two copies of a transcript on
two probes with the same UMI is extremely low. After reverse-transcription,
amplification, sequencing and alignment de-duplication can be performed by
identifying reads with the same UMI that align to the same position and
therefore should be PCR duplicates rather than truly expressed copies of a
transcript. For this method to be effective each read must be associated with a
UMI which means that only a small section at the 3’ end of each transcript is
sequenced. This has the side effect of reducing the amount of cDNA that needs
to be sequenced and therefore increasing the number of cells that can be
sequenced at a time. While the improvement in quantification of gene expression
levels is useful for many downstream analyses it comes at the cost of coverage
across the length of a gene which is required for applications such as variant
detection and de-novo assembly. Statistical methods designed for full-length
data may also be affected by the difference properties of a UMI dataset.
Datasets with UMIs also need extra processing steps which can be complicated by
the possibility of sequencing errors in the UMI itself.</p>
</div>
<div id="recent-advances" class="section level3">
<h3><span class="header-section-number">1.2.3</span> Recent advances</h3>
<ul>
<li>New capture methods</li>
<li>CITE-seq</li>
<li>Cell hashing</li>
<li>CRISPR</li>
<li>Multiple measurements, same cell</li>
</ul>
<p>Although droplet-based techniques are currently the most commonly used cell
capture technologies other approaches have been proposed that promise to capture
even more cells even more cheaply. These include approaches based around
nanowells…</p>
<p>Extensions to the standard protocols have also been proposed that allow extra
measurements from the same cell. One such protocol is CITE-seq which enables
measurement of the levels of selected proteins at the same time as the whole
transcriptome. Antibodies for the proteins of interest are labelled with short
nucleotide sequences. These antibodies can then be applied to the dissociated
cells and any that remain unbound washed away before cell capture. The antibody
labels are then captured along with mRNA transcripts and a size selection step
is applied to separate them before library preparation. Similar antibodies can
be used to allow multiplexing of samples through a process known as cell
hashing. In a typical scRNA-seq experiment each batch corresponds to a single
sample. This complicated analysis as it is impossible to tell what is noise due
to cells being processed in the same way and what is true biological signal.
Cell hashing uses an antibody to a ubiquitously expressed protein but with a
different nucleotide sequence for each sample. The samples can then be mixed,
processed in batches and then the cells computationally separated based on
which sequence they are associated with. An added benefit of this approach is
the simple detection of doublets containing cells from different samples.</p>
<p>CRISPR-Cas9 gene editing has also been developed as an extension to scRNA-seq
protocols. One possibility is to introduce a mutation at a known location that
can then be used to demultiplex samples processed together. It is possible to
do this with samples from different individuals or cell lines but the advantage
of a gene editing based approach is that the genetic background remains similar
between samples. It is also possible to investigate the effects of introducing a
mutation. Protocols like Perturb-Seq introduce a range of guide RNA molecules to
a cell culture, subject the cells to some stimulus then perform single-cell RNA
sequencing. The introduced mutation can then be linked to the response of the
cells to the stimulus and the associated broader changes in gene expression.</p>
<p>Other approaches that allow multiple measurements from individual cells
include…</p>
</div>
</div>
<div id="features-of-single-cell-rna-seq-data" class="section level2">
<h2><span class="header-section-number">1.3</span> Features of single-cell RNA-seq data</h2>
<ul>
<li>Why use single-cell?</li>
<li>Low counts
<ul>
<li>Dropout</li>
<li>Bursting</li>
<li>Biology</li>
</ul></li>
</ul>
</div>
<div id="analysing-scrna-seq-data" class="section level2">
<h2><span class="header-section-number">1.4</span> Analysing scRNA-seq data</h2>
<pre><code>Single-cell RNA-seq technologies have developed rapidly but there are still a
number of challenges. Existing protocols are inefficient, capturing only around
10 percent of transcripts in a cell[@Grun2014-zn]. Combined with the relatively
low sequencing depth that is commonly used, this results in a limited
sensitivity and an inability to reliably detect lowly expressed transcripts. The
small amount of starting material also contributes to high levels of technical
noise, complicating downstream analysis and making it difficult to detect
biological differences[@Liu2016-wq]. All high-throughput scRNA-seq protocols and
platforms require tissues to be dissociated into single-cell suspensions before
capture. This step can be non-trivial. Some tissues or cell types may not
readily separate and the treatments used to break them down may effect the
health of the cells and therefore their transcriptional profiles. Additionally,
some cell types may be too big or have other physical characteristics that
prevent them being captured using currect methods. Cells may also be damaged
during processing, or missing or multiple cells may have been sequenced, making
quality control an important consideration.

As well as introducing technical noise, the small amounts of starting material
and low sequencing depth mean there are many occasions  where there is no
measured expression for a particular gene in a particular cell. Some of these
zero counts are due to the biology we wish to study, for example we expect
different cell types to express different genes, but there are additional
biological factors such as the cell cycle, transcriptional bursting and
environmental interactions which cause genuine differences in expression between
cells performing the same function. On top on this are the technical effects
that have already been discussed including that  existing protocols may not
reliably capture all the RNA present, resulting in &quot;dropout&quot; events where a gene
is expressed in a sample but not observed in the sequencing data. These zeros
can make analysis difficult as methods must account for the missing information
and they may violate the assumptions of existing approaches. For example the
DEseq2 package[@Love2014-tw] has proven successful for detecting differential
expression in bulk RNA-seq but it relies on the presence of genes without any
zero counts, and therefore typically fails on scRNA-seq data. Examples of
scRNA-seq analysis packages that tackle this problem are Zero Inflated Factor
Analysis (ZIFA)[@Pierson2015-qp], which explicitly models dropout as it affects
dimensionality reduction, and Clustering through Imputation and Dimensionality
Reduction (CIDR)[@Lin2016-yu], which implicitly imputes zeros as it clusters
cells.

Bulk RNA-seq experiments are usually conducted on predefined groups of samples,
for example cancer cells and normal tissue, different tissue types or treated
and control cells. Some scRNA-seq experiments are done in a similar way where
cells are sorted into known types based on surface markers or selected at a
series of time points, but often they are more exploratory. Many of the current
studies have taken samples of developing or mature tissues and attempted to
identify what cell types are present[@Zeisel2015-rd; @Patel2014-bl;
@Treutlein2014-wd; @Usoskin2015-fz; @Buettner2015-rq; @Klein2015-iw;
@Trapnell2014-he]. This requires a new set of analysis techniques to be
developed which attempt to identify cell types. Success at this task is crucial
to the reliability of results from more developed analysis methods such as gene
testing. Additionally is it impossible to truly replicate an individual cell and
therefore thought must be given as to what constitutes a replicate for the
purposes of statistical analysis.</code></pre>
<div id="pre-processing-and-quality-control" class="section level3">
<h3><span class="header-section-number">1.4.1</span> Pre-processing and quality control</h3>
<ul>
<li>Alignment</li>
<li>Droplet selection</li>
<li>UMIs</li>
<li>Doublet detection</li>
<li>Bad cells</li>
<li>Gene filtering</li>
<li>Cell ranger</li>
<li>scater</li>
</ul>
<pre><code>As scRNA-seq data has become available there has been a rapid development of new
bioinformatics tools attempting to unlock its potential. Currently there are at
least 80 available software packages that have been designed specifically for
the analysis of scRNA-seq data, the majority of which have been published in
peer-reviewed journals or as preprints. A table of scRNA-seq software is
available at [https://goo.gl/4wcVwn](). Prior to analysis the sequencing reads
from an scRNA-seq experiment are processed in much the same way as a bulk
experiment. Typically there is some quality control of the raw reads, reads are
aligned to a reference genome and the number of reads overlapping annotated
features (genes or transcripts) is counted. Alternatively, for full-length
reads, probabilistic quantification methods such as kallisto[@Bray2016-tm] or
Salmon[@Patro2015-kl] can be used. These approaches can greatly  improve
processing time which is important when there may be tens of thousands of
samples but they are currently incompatible with UMI protocols. When using
conventional alignment UMI samples need extra processing with tools such as
UMI-tools[@Smith2016-bt] or umis[@Svensson2016-eg]. The resulting gene by cell
matrix of expression values is the starting point for most analysis.

Quality control of individual cells is important as most experiments will
contain some low-quality cells that could be uninformative or misleading.
Quality control can be performed on various levels: on the quality scores of the
reads themselves, how or where the reads align or features of the expression
matrix such as the total expression, expression of spike-ins or expression of
particular genes. The Cellity package attempts to do this by inspecting a series
of biological and technical features and using principal component analysis or
machine learning methods to distinguish between high and low-quality
cells[@Ilicic2016-wy]. However the authors found that many of the features were
cell type specific and more work needs to be done to make this approach more
generally applicable. Jiang, Thomson and Stewart take a different approach,
assuming that expression outliers are associated with poor sequencing
quality[@Jiang2016-ys]. The scater package[@McCarthy2016-cw] emphasises a more
exploratory approach to quality control. While it cannot automatically detect
low-quality cells, scater provides a convenient object for storing scRNA-seq
data with functions for plotting associated features, making it easy for the
user to define their own filtering thresholds. Plate-based platforms such as the
Fluidigm C1 can have additional biases based on the location of individual
wells. The OEFinder package attempts to identify and visualise these &quot;ordering
effects&quot;[@Leng2016-it].</code></pre>
</div>
<div id="integrating-multiple-datasets" class="section level3">
<h3><span class="header-section-number">1.4.2</span> Integrating multiple datasets</h3>
<ul>
<li>Why?</li>
<li>Seurat CCA</li>
<li>New methods</li>
</ul>
<pre><code>Technical variation is a known problem in high-throughput studies and Kim et al.
predict that only 17.8 percent of allele-specific expression is due to
biological variation with the rest being technical noise[@Kim2015-mo]. Effective
normalisation has been shown to be a crucial aspect of analysis for bulk RNA-seq
datasets, but how normalisation should be be applied to single-cell datasets is
yet to be clearly established. Some studies use simple transformations like
Reads (or Fragments) Per Kilobase per Million (RPKM/FPKM)[@Mortazavi2008-vu] or
Transcripts Per Million (TPM)[@Wagner2012-qf] which correct for library size and
gene length. Alternatively, normalisation methods designed for detecting
differential expression in bulk samples such as the Trimmed Mean of M-Values
(TMM)[@Robinson2010-ll] or the DESeq method[@Anders2010-pq] can be applied, but
it is unclear how suitable they are for the single-cell context. Most of the
methods that have been developed specifically for estimating technical variance
in scRNA-seq data make use of spike-ins. Brennecke et al.[@Brennecke2013-pt],
Ding et al.[@Ding2015-ht] and Grün, Kester and van Oudenaarden[@Grun2014-zn] all
propose methods for estimating technical variance using spike-ins, as does
Bayesian Analysis of Single-Cell Sequencing data (BASiCS)[@Vallejos2015-ef].
However, using spike-ins for normalisation relys on the assumption that they
properly capture the dynamics of the underlying datasets, and even if this is
the case is it restrictive as they are not compatible with all current
sequencing protocols. Lun, Bach and Marioni don&#39;t make use of spike-ins, instead
using a pooling approach to compensate for the large number of zero counts,
where expression levels are summed across similar cells before calculating
normalisation factors that are deconvolved back to the individual cell
level[@Lun2016-mq].</code></pre>
</div>
<div id="grouping-cells" class="section level3">
<h3><span class="header-section-number">1.4.3</span> Grouping cells</h3>
<ul>
<li>Clustering</li>
<li>Seurat</li>
<li>Other approaches</li>
</ul>
<pre><code>Once a set of high-quality cells has been established the true analysis can
begin. Many of the current packages focus on the task of assigning cells to
groups before applying more traditional differential expression testing. This
approach makes sense for a sample with a defined set of mature cell types and is
taken by tools such as SINgle CEll RNA-seq profiling Analysis
(SINCERA)[@Guo2015-mf], Single-Cell Consensus Clustering (SC3)[@Kiselev2016-fa],
Seurat[@Satija2015-or], single-cell latent variable model
(scLVM)[@Buettner2015-rq] and Spanning-tree Progression Analysis of
Density-normalised Events (SPADE)[@Anchang2016-vo], as well as BackSPIN which
was used to identify nine cell types and 47 distinct subclasses in the mouse
cortex and hippocampus[@Zeisel2015-rd]. These tools attempt to cluster similar
cells together based on their expression profiles, forming groups of cells of
the same type. Often a dimensionality reduction step is included which can help
to remove some of the noise present in scRNA-seq data. Once groups of cells are
identified many of these packages can test genes for changes in expression,
identifying genes that are differentially expressed across the groups, or marker
genes that are expressed in a single group. These genes can be used to identify
which cell types each group represents, or alternatively known marker genes can
be inspected.</code></pre>
</div>
<div id="ordering-cells" class="section level3">
<h3><span class="header-section-number">1.4.4</span> Ordering cells</h3>
<ul>
<li>Pseudotime</li>
<li>Monocle</li>
<li>Other approaches</li>
</ul>
<pre><code>In other situations, for example where stem cells are differentiating into
mature cell types, it may be more appropriate to order cells along a continuous
trajectory from one cell type to another. Trajectory analysis was pioneered by
Monocle which used dimensionality reduction and computation of a minimum
spanning tree to explore a model of skeletal muscle
differentiation[@Trapnell2014-he]. Since then the Monocle algorithm has been
updated and a range of others developed, including TSCAN[@Ji2016-ws],
SLICER[@Welch2016-cw], CellTree[@DuVerle2016-ni], Sincell[@Julia2015-zc] and
Mpath[@Chen2016-kx]. In their review of methods for trajectory inference,
Cannoodt, Saelens and Saeys break the process into two parts, dimensionality
reduction and then trajectory modelling (Figure
\@ref(fig:trajectory-inference))[@Cannoodt2016-iv]. Dimensionality reduction
consists of calculating similarities between cells, projecting onto lower
dimensions using manifold learning techniques such as PCA or
t-SNE[@Maaten2008-ne] then clustering cells or constructing a graph between
then. The trajectory is then formed by finding a path between cells and ordering
the cells along it. Once a path has been inferred important genes can be
identified by looking for those that change expression over the course of the
path. These genes can be important in their own right as they describe the
biology of the path, but that can also be used to identify cell types at the end
points of the path or where the path branches. Deciding which assignment
approach is most appropriate depends on the source of that data and the
questions you intend to ask. There are currently no studies comprehensively
comparing the performance of different methods for each approach.</code></pre>
</div>
<div id="gene-detection-and-interpretation" class="section level3">
<h3><span class="header-section-number">1.4.5</span> Gene detection and interpretation</h3>
<ul>
<li>DE</li>
<li>Marker genes
<ul>
<li>Alternatives - Gini, classifiers</li>
</ul></li>
<li>Reviews</li>
<li>Classification</li>
</ul>
</div>
</div>
<div id="kidney-development" class="section level2">
<h2><span class="header-section-number">1.5</span> Kidney development</h2>
<div id="structure-and-function" class="section level3">
<h3><span class="header-section-number">1.5.1</span> Structure and function</h3>
<ul>
<li>Kidney structure</li>
<li>Nephron structure</li>
<li>Important cell types</li>
</ul>
</div>
<div id="stages-of-development" class="section level3">
<h3><span class="header-section-number">1.5.2</span> Stages of development</h3>
<ul>
<li>Lineage</li>
<li>Important genes</li>
</ul>
</div>
<div id="growing-kidney-organoids" class="section level3">
<h3><span class="header-section-number">1.5.3</span> Growing kidney organoids</h3>
<ul>
<li>Why?
<ul>
<li>Disease modelling</li>
</ul></li>
<li>Protocol</li>
<li>Growth factors</li>
<li>Characterisation</li>
<li>Reproducibility</li>
</ul>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="2-the-scrna-seq-tools-landscape.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
