<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Tools and techniques for single-cell RNA-seq data</title>
  <meta name="description" content="Tools and techniques for single-cell RNA-seq data">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Tools and techniques for single-cell RNA-seq data" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Tools and techniques for single-cell RNA-seq data" />
  
  
  

<meta name="author" content="Luke Zappia">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="3-simulating-scrna-seq-data.html">
<link rel="next" href="5-analysis-of-kidney-organoid-scrna-seq-data.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Front matter</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#declaration"><i class="fa fa-check"></i>Declaration</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#preface"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#rna-sequencing"><i class="fa fa-check"></i><b>1.1</b> RNA sequencing</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-introduction.html"><a href="1-introduction.html#library-preparation"><i class="fa fa-check"></i><b>1.1.1</b> Library preparation</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-introduction.html"><a href="1-introduction.html#high-throughput-sequencing"><i class="fa fa-check"></i><b>1.1.2</b> High-throughput sequencing</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-introduction.html"><a href="1-introduction.html#analysis-of-rna-seq-data"><i class="fa fa-check"></i><b>1.1.3</b> Analysis of RNA-seq data</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#single-cell-rna-sequencing"><i class="fa fa-check"></i><b>1.2</b> Single-cell RNA-sequencing</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#single-cell-capture-technologies"><i class="fa fa-check"></i><b>1.3</b> Single-cell capture technologies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-introduction.html"><a href="1-introduction.html#droplet-based-cell-capture"><i class="fa fa-check"></i><b>1.3.1</b> Droplet based cell capture</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introduction.html"><a href="1-introduction.html#unique-molecular-identifiers"><i class="fa fa-check"></i><b>1.3.2</b> Unique Molecular Identifiers</a></li>
<li class="chapter" data-level="1.3.3" data-path="1-introduction.html"><a href="1-introduction.html#recent-advances"><i class="fa fa-check"></i><b>1.3.3</b> Recent advances</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#analysing-scrna-seq-data"><i class="fa fa-check"></i><b>1.4</b> Analysing scRNA-seq data</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-introduction.html"><a href="1-introduction.html#pre-processing-and-quality-control"><i class="fa fa-check"></i><b>1.4.1</b> Pre-processing and quality control</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-introduction.html"><a href="1-introduction.html#normalisation-and-integration"><i class="fa fa-check"></i><b>1.4.2</b> Normalisation and integration</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-introduction.html"><a href="1-introduction.html#grouping-cells"><i class="fa fa-check"></i><b>1.4.3</b> Grouping cells</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-introduction.html"><a href="1-introduction.html#ordering-cells"><i class="fa fa-check"></i><b>1.4.4</b> Ordering cells</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-introduction.html"><a href="1-introduction.html#gene-detection-and-interpretation"><i class="fa fa-check"></i><b>1.4.5</b> Gene detection and interpretation</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-introduction.html"><a href="1-introduction.html#alternative-analyses"><i class="fa fa-check"></i><b>1.4.6</b> Alternative analyses</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-introduction.html"><a href="1-introduction.html#kidney-development"><i class="fa fa-check"></i><b>1.5</b> Kidney development</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-introduction.html"><a href="1-introduction.html#structure-and-function"><i class="fa fa-check"></i><b>1.5.1</b> Structure and function</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-introduction.html"><a href="1-introduction.html#stages-of-development"><i class="fa fa-check"></i><b>1.5.2</b> Stages of development</a></li>
<li class="chapter" data-level="1.5.3" data-path="1-introduction.html"><a href="1-introduction.html#growing-kidney-organoids"><i class="fa fa-check"></i><b>1.5.3</b> Growing kidney organoids</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-the-scrna-seq-tools-landscape.html"><a href="2-the-scrna-seq-tools-landscape.html"><i class="fa fa-check"></i><b>2</b> The scRNA-seq tools landscape</a><ul>
<li class="chapter" data-level="2.1" data-path="2-the-scrna-seq-tools-landscape.html"><a href="2-the-scrna-seq-tools-landscape.html#introduction-1"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-simulating-scrna-seq-data.html"><a href="3-simulating-scrna-seq-data.html"><i class="fa fa-check"></i><b>3</b> Simulating scRNA-seq data</a><ul>
<li class="chapter" data-level="3.1" data-path="3-simulating-scrna-seq-data.html"><a href="3-simulating-scrna-seq-data.html#introduction-2"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-simulating-scrna-seq-data.html"><a href="3-simulating-scrna-seq-data.html#splatter-publication"><i class="fa fa-check"></i><b>3.2</b> Splatter publication</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-visualising-clustering-across-resolutions.html"><a href="4-visualising-clustering-across-resolutions.html"><i class="fa fa-check"></i><b>4</b> Visualising clustering across resolutions</a><ul>
<li class="chapter" data-level="4.1" data-path="4-visualising-clustering-across-resolutions.html"><a href="4-visualising-clustering-across-resolutions.html#introduction-3"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-analysis-of-kidney-organoid-scrna-seq-data.html"><a href="5-analysis-of-kidney-organoid-scrna-seq-data.html"><i class="fa fa-check"></i><b>5</b> Analysis of kidney organoid scRNA-seq data</a></li>
<li class="chapter" data-level="6" data-path="6-conclusion.html"><a href="6-conclusion.html"><i class="fa fa-check"></i><b>6</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Tools and techniques for single-cell RNA-seq data</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="visualising-clustering-across-resolutions" class="section level1">
<h1><span class="header-section-number">Chapter 4</span> Visualising clustering across resolutions</h1>
<div id="introduction-3" class="section level2">
<h2><span class="header-section-number">4.1</span> Introduction</h2>
<p>Clustering of cells to form groups is a common task when analysing scRNA-seq
data that is not required for bulk RNA-seq experiments and one that has
received a lot of attention from analysis methods developers. The need to group
samples is not unique to genomic data and clustering techniques are used in
many other fields for a wide variety of purposes. Whatever kind of data you are
interested in and whatever clustering method is being used a question that
commonly comes up is how many clusters do we want to have? This can be
controlled by setting an exact value, changing a parameter that indirectly
controls the clustering resolution or affected by the values of other
parameters and the number of clusters that are used can often have a profound
affect on how the results are interpreted. Existing measures of clustering
typically only consider a single clustering resolution at a time or require
multiple rounds or permutations and clustering which can be infeasible for large
datasets. In this chapter I propose an alternative visualisation-based aid for
deciding which clustering resolution to use.</p>
<p>Clusterings of the same dataset at different resolutions are often related and
it is common for new clusters formed at higher resolutions to be formed by
splitting existing clusters. However when comparing clusterings it is not
always clear what those relationships are and how significant they might be.
The method I describe here was published in <em>GigaScience</em> and suggests
clustering datasets at multiple resolutions then considering the overlap in
samples at neighbouring resolutions. By doing this we can build a graph
structure we call a “clustering tree”. Visualising this tree allows us to see
where new clusters form, how they are related and the stability of particular
clustering resolutions. In the paper we demonstrate this approach using
simulated datasets, a simple dataset commonly used as an example for machine
learning techniques and a complex scRNA-seq dataset from blood.</p>
<p>While the structure of clustering trees can help choose a clustering
resolutions to use they are more generally a compact, information dense
visualisation that can show information across clustering resolutions. This is
something that is not possible with traditional visualisations used for
clustering results such as t-SNE projections and is achieved by trading
individual information about each sample for summarised information about
clusters and adding a resolution dimension. Overlaying important domain
knowledge (such as the expression of known marker genes) onto these
visualisations can be particularly informative and we also demonstrate this in
our paper.</p>
<p>Clustering trees can be produced using the clustree R package which is built on
the tidygraph and ggraph packages and is available from CRAN
(<a href="https://cran.r-project.org/package=clustree" class="uri">https://cran.r-project.org/package=clustree</a>).</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="3-simulating-scrna-seq-data.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="5-analysis-of-kidney-organoid-scrna-seq-data.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
