<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Tools and techniques for single-cell RNA-seq data</title>
  <meta name="description" content="Tools and techniques for single-cell RNA-seq data">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Tools and techniques for single-cell RNA-seq data" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Tools and techniques for single-cell RNA-seq data" />
  
  
  

<meta name="author" content="Luke Zappia">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="5-analysis-of-kidney-organoid-scrna-seq-data.html">
<link rel="next" href="references.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Front matter</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#declaration"><i class="fa fa-check"></i>Declaration</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#preface"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#rna-sequencing"><i class="fa fa-check"></i><b>1.1</b> RNA sequencing</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-introduction.html"><a href="1-introduction.html#library-preparation"><i class="fa fa-check"></i><b>1.1.1</b> Library preparation</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-introduction.html"><a href="1-introduction.html#high-throughput-sequencing"><i class="fa fa-check"></i><b>1.1.2</b> High-throughput sequencing</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-introduction.html"><a href="1-introduction.html#analysis-of-rna-seq-data"><i class="fa fa-check"></i><b>1.1.3</b> Analysis of RNA-seq data</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#single-cell-rna-sequencing"><i class="fa fa-check"></i><b>1.2</b> Single-cell RNA-sequencing</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#single-cell-capture-technologies"><i class="fa fa-check"></i><b>1.3</b> Single-cell capture technologies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-introduction.html"><a href="1-introduction.html#droplet-based-cell-capture"><i class="fa fa-check"></i><b>1.3.1</b> Droplet based cell capture</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introduction.html"><a href="1-introduction.html#unique-molecular-identifiers"><i class="fa fa-check"></i><b>1.3.2</b> Unique Molecular Identifiers</a></li>
<li class="chapter" data-level="1.3.3" data-path="1-introduction.html"><a href="1-introduction.html#recent-advances"><i class="fa fa-check"></i><b>1.3.3</b> Recent advances</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#analysing-scrna-seq-data"><i class="fa fa-check"></i><b>1.4</b> Analysing scRNA-seq data</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-introduction.html"><a href="1-introduction.html#pre-processing-and-quality-control"><i class="fa fa-check"></i><b>1.4.1</b> Pre-processing and quality control</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-introduction.html"><a href="1-introduction.html#normalisation-and-integration"><i class="fa fa-check"></i><b>1.4.2</b> Normalisation and integration</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-introduction.html"><a href="1-introduction.html#grouping-cells"><i class="fa fa-check"></i><b>1.4.3</b> Grouping cells</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-introduction.html"><a href="1-introduction.html#ordering-cells"><i class="fa fa-check"></i><b>1.4.4</b> Ordering cells</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-introduction.html"><a href="1-introduction.html#gene-detection-and-interpretation"><i class="fa fa-check"></i><b>1.4.5</b> Gene detection and interpretation</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-introduction.html"><a href="1-introduction.html#alternative-analyses"><i class="fa fa-check"></i><b>1.4.6</b> Alternative analyses</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-introduction.html"><a href="1-introduction.html#kidney-development"><i class="fa fa-check"></i><b>1.5</b> Kidney development</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-introduction.html"><a href="1-introduction.html#structure-and-function"><i class="fa fa-check"></i><b>1.5.1</b> Structure and function</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-introduction.html"><a href="1-introduction.html#stages-of-development"><i class="fa fa-check"></i><b>1.5.2</b> Stages of development</a></li>
<li class="chapter" data-level="1.5.3" data-path="1-introduction.html"><a href="1-introduction.html#growing-kidney-organoids"><i class="fa fa-check"></i><b>1.5.3</b> Growing kidney organoids</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-the-scrna-seq-tools-landscape.html"><a href="2-the-scrna-seq-tools-landscape.html"><i class="fa fa-check"></i><b>2</b> The scRNA-seq tools landscape</a><ul>
<li class="chapter" data-level="2.1" data-path="2-the-scrna-seq-tools-landscape.html"><a href="2-the-scrna-seq-tools-landscape.html#introduction-1"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-simulating-scrna-seq-data.html"><a href="3-simulating-scrna-seq-data.html"><i class="fa fa-check"></i><b>3</b> Simulating scRNA-seq data</a><ul>
<li class="chapter" data-level="3.1" data-path="3-simulating-scrna-seq-data.html"><a href="3-simulating-scrna-seq-data.html#introduction-2"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-simulating-scrna-seq-data.html"><a href="3-simulating-scrna-seq-data.html#splatter-publication"><i class="fa fa-check"></i><b>3.2</b> Splatter publication</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-visualising-clustering-across-resolutions.html"><a href="4-visualising-clustering-across-resolutions.html"><i class="fa fa-check"></i><b>4</b> Visualising clustering across resolutions</a><ul>
<li class="chapter" data-level="4.1" data-path="4-visualising-clustering-across-resolutions.html"><a href="4-visualising-clustering-across-resolutions.html#introduction-3"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-analysis-of-kidney-organoid-scrna-seq-data.html"><a href="5-analysis-of-kidney-organoid-scrna-seq-data.html"><i class="fa fa-check"></i><b>5</b> Analysis of kidney organoid scRNA-seq data</a></li>
<li class="chapter" data-level="6" data-path="6-conclusion.html"><a href="6-conclusion.html"><i class="fa fa-check"></i><b>6</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Tools and techniques for single-cell RNA-seq data</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="conclusion" class="section level1">
<h1><span class="header-section-number">Chapter 6</span> Conclusion</h1>
<ul>
<li>What is in this thesis?
<ul>
<li>tools</li>
<li>simulation</li>
<li>clustering trees</li>
<li>organoid analysis</li>
<li>tools etc.</li>
</ul></li>
<li>Where is the field going?
<ul>
<li>more cells</li>
<li>references</li>
<li>new technologies</li>
</ul></li>
</ul>
<p>This thesis explores the tools and techniques used for analysing single-cell
RNA sequencing data. This data has been made available by the recent
development of technologies that allow individual cells to be captured, RNA
from them isolated and processed and the expression levels of genes to be
measured. This new source of data opens up a range of possibilities but it also
presents a new set of challenges and the computational analyse of it is still a
developing field. My contribution to this area consists of four discrete but
related projects. In chapter X I present my work developing and maintaining a
database of software tools for analysing scRNA-seq data. This database allows
to identify focuses of methods development, features of how tools are developed
and shared and display trends other time. This information is made available to
the community via the scRNA-tools website. Chapter X describes Splatter a
software package for generating synthetic scRNA-seq dataset. Simulations are an
important of evaluating computational methods and Splatter provides a
consistent, easy-to-use interface to several published simulation models as
well as functions for assessing their similarity to real scRNA-seq datasets.
Methods for clustering scRNA-seq datasets to form groups of cells with similar
transcriptional profiles has been a particular focus of methods developers and
most of these approaches have parameters than affect how many clusters are
produced. The number of clusters can influence how the results are interpreted
but it is difficult to choose these parameters. In chapter X I describe an
algorithm that uses clusterings at multiple resolutions to produce a
visualisation that can help with selecting clustering parameters. A software
package for producing these visualisations is available and they can be used
more generally from displaying information across the resolution dimension. The
final project of my thesis applies analysis techniques to an scRNA-seq dataset
from kidney organoids in order to identify and characterise the cell types
present.</p>
<p>Although scRNA-seq technologies have been only been available for a short time
they continue to develop. New platforms and protocols are been continually
developed which promise to create even larger and more detailed datasets. As
these become available more computational challenges will emerge both in how to
extract information from them and how to manage the computational workload.
Other measurements from individual cells are also becoming available, such as
measuring chromatin accessibility using single-cell ATAC-seq or spatial
transcriptomics techniques. Learning how to properly integrate these different
types of data will need to another area of significant research. Deciding which
methods to use for various tasks will require significant effort at
benchmarking and evaluation but is vital for making the most of this data. Even
when scRNA-seq analysis methods are reliable and effective interpreting their
results can be difficult. Large scale project such has the Human Cell Atlas aim
to produce reference datasets to make this easier and allow different analysis
approaches but this is an extremely difficult task that will take many years
and a well-coordinated global effort. We are just at the start of this process
and while it looks promising at this stage it will always be important to work
closely with domain expert in the system that is being studied. Exploratory
analyses have already been completed on most tissue types and many researchers
will begin to turn their attention to more designed experiments that aim to
answer specific questions. Over the next few years I expect the development of
new technologies and computational methods continue at the same pace, hopefully
converging on a set of tools that work well for standard analyses. Single-cell
RNA sequencing is an exciting technology that has opened up a world of
possibilities but we are still at the early stages of working out how best to
use it.</p>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="5-analysis-of-kidney-organoid-scrna-seq-data.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
