<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Tools and techniques for single-cell RNA-seq data</title>
  <meta name="description" content="Tools and techniques for single-cell RNA-seq data">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Tools and techniques for single-cell RNA-seq data" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Tools and techniques for single-cell RNA-seq data" />
  
  
  

<meta name="author" content="Luke Zappia">



  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  

<link rel="next" href="1-introduction.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./"></a></li>
<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Front matter</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#declaration"><i class="fa fa-check"></i>Declaration</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#preface"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="1-introduction.html"><a href="1-introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="1-introduction.html"><a href="1-introduction.html#rna-sequencing"><i class="fa fa-check"></i><b>1.1</b> RNA sequencing</a><ul>
<li class="chapter" data-level="1.1.1" data-path="1-introduction.html"><a href="1-introduction.html#library-preparation"><i class="fa fa-check"></i><b>1.1.1</b> Library preparation</a></li>
<li class="chapter" data-level="1.1.2" data-path="1-introduction.html"><a href="1-introduction.html#high-throughput-sequencing"><i class="fa fa-check"></i><b>1.1.2</b> High-throughput sequencing</a></li>
<li class="chapter" data-level="1.1.3" data-path="1-introduction.html"><a href="1-introduction.html#analysis-of-rna-seq-data"><i class="fa fa-check"></i><b>1.1.3</b> Analysis of RNA-seq data</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="1-introduction.html"><a href="1-introduction.html#single-cell-rna-sequencing"><i class="fa fa-check"></i><b>1.2</b> Single-cell RNA-sequencing</a></li>
<li class="chapter" data-level="1.3" data-path="1-introduction.html"><a href="1-introduction.html#single-cell-capture-technologies"><i class="fa fa-check"></i><b>1.3</b> Single-cell capture technologies</a><ul>
<li class="chapter" data-level="1.3.1" data-path="1-introduction.html"><a href="1-introduction.html#droplet-based-cell-capture"><i class="fa fa-check"></i><b>1.3.1</b> Droplet based cell capture</a></li>
<li class="chapter" data-level="1.3.2" data-path="1-introduction.html"><a href="1-introduction.html#unique-molecular-identifiers"><i class="fa fa-check"></i><b>1.3.2</b> Unique Molecular Identifiers</a></li>
<li class="chapter" data-level="1.3.3" data-path="1-introduction.html"><a href="1-introduction.html#recent-advances"><i class="fa fa-check"></i><b>1.3.3</b> Recent advances</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="1-introduction.html"><a href="1-introduction.html#analysing-scrna-seq-data"><i class="fa fa-check"></i><b>1.4</b> Analysing scRNA-seq data</a><ul>
<li class="chapter" data-level="1.4.1" data-path="1-introduction.html"><a href="1-introduction.html#pre-processing-and-quality-control"><i class="fa fa-check"></i><b>1.4.1</b> Pre-processing and quality control</a></li>
<li class="chapter" data-level="1.4.2" data-path="1-introduction.html"><a href="1-introduction.html#normalisation-and-integration"><i class="fa fa-check"></i><b>1.4.2</b> Normalisation and integration</a></li>
<li class="chapter" data-level="1.4.3" data-path="1-introduction.html"><a href="1-introduction.html#grouping-cells"><i class="fa fa-check"></i><b>1.4.3</b> Grouping cells</a></li>
<li class="chapter" data-level="1.4.4" data-path="1-introduction.html"><a href="1-introduction.html#ordering-cells"><i class="fa fa-check"></i><b>1.4.4</b> Ordering cells</a></li>
<li class="chapter" data-level="1.4.5" data-path="1-introduction.html"><a href="1-introduction.html#gene-detection-and-interpretation"><i class="fa fa-check"></i><b>1.4.5</b> Gene detection and interpretation</a></li>
<li class="chapter" data-level="1.4.6" data-path="1-introduction.html"><a href="1-introduction.html#alternative-analyses"><i class="fa fa-check"></i><b>1.4.6</b> Alternative analyses</a></li>
</ul></li>
<li class="chapter" data-level="1.5" data-path="1-introduction.html"><a href="1-introduction.html#kidney-development"><i class="fa fa-check"></i><b>1.5</b> Kidney development</a><ul>
<li class="chapter" data-level="1.5.1" data-path="1-introduction.html"><a href="1-introduction.html#structure-and-function"><i class="fa fa-check"></i><b>1.5.1</b> Structure and function</a></li>
<li class="chapter" data-level="1.5.2" data-path="1-introduction.html"><a href="1-introduction.html#stages-of-development"><i class="fa fa-check"></i><b>1.5.2</b> Stages of development</a></li>
<li class="chapter" data-level="1.5.3" data-path="1-introduction.html"><a href="1-introduction.html#growing-kidney-organoids"><i class="fa fa-check"></i><b>1.5.3</b> Growing kidney organoids</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="2-the-scrna-seq-tools-landscape.html"><a href="2-the-scrna-seq-tools-landscape.html"><i class="fa fa-check"></i><b>2</b> The scRNA-seq tools landscape</a><ul>
<li class="chapter" data-level="2.1" data-path="2-the-scrna-seq-tools-landscape.html"><a href="2-the-scrna-seq-tools-landscape.html#introduction-1"><i class="fa fa-check"></i><b>2.1</b> Introduction</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="3-simulating-scrna-seq-data.html"><a href="3-simulating-scrna-seq-data.html"><i class="fa fa-check"></i><b>3</b> Simulating scRNA-seq data</a><ul>
<li class="chapter" data-level="3.1" data-path="3-simulating-scrna-seq-data.html"><a href="3-simulating-scrna-seq-data.html#introduction-2"><i class="fa fa-check"></i><b>3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.2" data-path="3-simulating-scrna-seq-data.html"><a href="3-simulating-scrna-seq-data.html#splatter-publication"><i class="fa fa-check"></i><b>3.2</b> Splatter publication</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="4-visualising-clustering-across-resolutions.html"><a href="4-visualising-clustering-across-resolutions.html"><i class="fa fa-check"></i><b>4</b> Visualising clustering across resolutions</a><ul>
<li class="chapter" data-level="4.1" data-path="4-visualising-clustering-across-resolutions.html"><a href="4-visualising-clustering-across-resolutions.html#introduction-3"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="5-analysis-of-kidney-organoid-scrna-seq-data.html"><a href="5-analysis-of-kidney-organoid-scrna-seq-data.html"><i class="fa fa-check"></i><b>5</b> Analysis of kidney organoid scRNA-seq data</a></li>
<li class="chapter" data-level="6" data-path="6-conclusion.html"><a href="6-conclusion.html"><i class="fa fa-check"></i><b>6</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Tools and techniques for single-cell RNA-seq data</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Tools and techniques for single-cell RNA-seq data</h1>
<h4 class="author"><em>Luke Zappia</em></h4>
</div>
<div id="front-matter" class="section level1 unnumbered">
<h1>Front matter</h1>

<!-- This document is used to add the front matter to formats other than LaTeX
(mostly word). These sections can be removed if you don't want them or the whole
file excluded from _bookdown.yml.--->
<div id="abstract" class="section level2 unnumbered">
<h2>Abstract</h2>
The preface pretty much says it all.

<p>Second paragraph of abstract starts here.</p>
</div>
<div id="declaration" class="section level2 unnumbered">
<h2>Declaration</h2>
<p>This is to certify that:</p>
<ol style="list-style-type: lower-roman">
<li>the thesis comprises only their original work towards the [name of
the award] except where indicated in the preface;</li>
<li>due acknowledgement has been made in the text to all other material
used; and</li>
<li>the thesis is fewer than the maximum word limit in length, exclusive of
tables, maps, bibliographies and appendices or that the thesis is [number
of words] as approved by the Research Higher Degrees Committee.</li>
</ol>
<p><!-- Signature line will be placed here --></p>
</div>
<div id="preface" class="section level2 unnumbered">
<h2>Preface</h2>
<ul>
<li>Chapter order</li>
<li>Publications</li>
<li>Contributions</li>
</ul>
<p>This preface provides a summary of the chapters in this thesis and described my
contribution to them. This is a thesis <em>with</em> publication and where
publications form part of a chapter that are listed here. Publications are
included as they appear online and are designed to be read as stand-alone
documents. Sections within these publications are not included in the table of
contents and references are available at the end of each publication rather
than the reference list for this thesis. These publications have authors other
than myself and their contributions are explained below. I am the first author
on these publications and contributed more than 50 percent of the work towards
them including drafting, editing and revising the manuscripts. My co-authors
have provided signed declarations acknowledging and supporting my contributions
which have been submitted along with this thesis. Where publicly available
datasets have been used these have been appropriately cited.</p>
<p><strong>Chapter 1</strong> is an original work providing a background and overview relevant
to understanding my work in this thesis including an introduction to RNA
sequencing, single-cell RNA sequencing and kidney function and development.</p>
<p><strong>Chapter 2</strong> is an original work describing a database of tools for analysing
single-cell RNA sequencing data which has been published in <em>PLoS Computational
Biology</em> as <em>“Exploring the single-cell RNA-seq analysis landscape with the
scRNA-tools database”</em>. In addition to the publication I developed a website
displaying the information in this database available at
<a href="https://scRNA-tools.org" class="uri">https://scRNA-tools.org</a>. The database and code for building the website is
available on GitHub at <a href="https://github.com/Oshlack/scRNA-tools" class="uri">https://github.com/Oshlack/scRNA-tools</a> under an MIT
license.</p>
<p>Contributions to the work in this chapter:</p>
<ul>
<li>I complied and regularly updated the database of tools.</li>
<li>I designed and built the public website used to display the database. Breon
Schmidt provided assistance with implementing some of the website
functionality and code for processing the database was based on a script
written by Sean Davis.</li>
<li>I performed the analysis of the database presented in the publication.</li>
<li>I wrote the first draft of the manuscript and produced all the figures in the
publication.</li>
<li>Alicia Oshlack provided advice on planning the manuscript and edited draft
versions.</li>
<li>Belinda Phipson contributed to writing the manuscript.</li>
</ul>
<p><strong>Chapter 3</strong> is an original work describing a software package for simulating
single-cell RNA sequencing data. This work was published in <em>Genome Biology</em> as
<em>“Splatter: simulation of single-cell RNA sequencing data”</em>. The software
package described in this publication is available through Bioconductor at
<a href="https://bioconductor.org/packages/splatter" class="uri">https://bioconductor.org/packages/splatter</a> and the code is shared on GitHub
at <a href="https://github.com/Oshlack/splatter" class="uri">https://github.com/Oshlack/splatter</a> under a GPL-3.0 license.</p>
<p>Contributions to the work in this chapter:</p>
<ul>
<li>I designed and implemented the Splatter R package described in this chapter</li>
<li>Belinda Phipson contributed to the design of the Splat simulation method
described in the publication and provided statistical advice.</li>
<li>I conducted the analysis presented in the publication and produced the figures
shown.</li>
<li>Belinda Phipson performed preprocessing for some of the public datasets used</li>
<li>Alicia Oshlack helped to design and plan the analysis presented in the
publication.</li>
<li>I wrote the first draft of the manuscript and performed revisions.</li>
<li>Alicia Oshlack assisted with planning the manuscript and edited drafts.</li>
<li>Belinda Phipson helped write sections of the manuscript and edited drafts.</li>
<li>Jovana Maksimovic proofread a section of the manuscript and provided comments.</li>
</ul>
<p><strong>Chapter 4</strong> is an original work describing a visualisation for showing
clustering results across multiple resolutions and helping to choose a
clustering resolution to use. This work has been published in <em>GigaScience</em> as
<em>“Clustering trees: a visualization for evaluating clusterings at multiple
resolutions”</em> and a software package implementing the algorithm described is
available from CRAN at <a href="https://cran.r-project.org/package=clustree" class="uri">https://cran.r-project.org/package=clustree</a>. The source
code for this package can be found on GitHub at
<a href="https://github.com/lazappi/clustree" class="uri">https://github.com/lazappi/clustree</a> under a GPL-3.0 license.</p>
<p>Contributions to work in this chapter:</p>
<ul>
<li>I designed the clustering trees algorithm described in this chapter.</li>
<li>I designed and wrote the clustree R package that implements this algorithm.</li>
<li>I performed the analysis presented in the manuscript and designed and produced
the figures shown.</li>
<li>Alicia Oshlack provided advice on the design and planning of the analysis to
present.</li>
<li>I planned and wrote the first draft of the manscript.</li>
<li>Alicia Oshlack provided advice on the structure of the manuscript and edited
draft versions.</li>
<li>I performed revisions and drafted responses to reviewers.</li>
<li>Marek Cmero read and provided comments on a draft of the manuscript.</li>
</ul>
<p><strong>Chapter 5</strong> is an original work where I performed analysis of a single-cell
RNA sequencing experiment from kidney organoids in order to identify and
characterise the cell types present.</p>
<p>Contributions to work in this chapter:</p>
<ul>
<li>XXX performed the cell culture.</li>
<li>XXX performed cell capture and library preparation of the samples.</li>
<li>I performed preprocessing of the datasets</li>
<li>I designed and performed the analysis with some input from Alicia Oshlack,
Belinda Phipson, Melissa Little and Alex Combes.</li>
<li>Alex Combes helped with interpreting gene lists describing cell types.</li>
<li>I designed and created the figures shown in this chapter.</li>
</ul>
<p><strong>Chapter 6</strong> is an original work summarising the work in this thesis, placing
it in the wider context of single-cell RNA sequencing analysis and outlining
potential directions of the field.</p>
<p><strong>Other publications I have contributed to during my candidature but are not
presented in this thesis</strong></p>
<ul>
<li>Organoid scRNA-seq paper</li>
<li>Gene length paper</li>
<li>Swirler organoid paper</li>
<li>Reproducibility paper</li>
</ul>
</div>
<div id="acknowledgements" class="section level2 unnumbered">
<h2>Acknowledgements</h2>
<p>This template is based on thesisdown (<a href="https://github.com/ismayc/thesisdown" class="uri">https://github.com/ismayc/thesisdown</a>) and
makes use of RMarkdown (<a href="https://rmarkdown.rstudio.com/" class="uri">https://rmarkdown.rstudio.com/</a>) and bookdown
<a href="https://bookdown.org/yihui/bookdown/" class="uri">https://bookdown.org/yihui/bookdown/</a>. The LaTeX template is based on John
Papandriopoulos’ University of Melbourne thesis template
(<a href="https://github.com/jpap/phd-thesis-template" class="uri">https://github.com/jpap/phd-thesis-template</a>). Inspriation also comes from
similar projects including beaverdown (<a href="https://github.com/zkamvar/beaverdown" class="uri">https://github.com/zkamvar/beaverdown</a>),
aggidown (<a href="https://github.com/ryanpeek/aggiedown" class="uri">https://github.com/ryanpeek/aggiedown</a>),
huskydown (<a href="https://github.com/benmarwick/huskydown" class="uri">https://github.com/benmarwick/huskydown</a>) and jayhawkdown
(<a href="https://github.com/wjakethompson/jayhawkdown" class="uri">https://github.com/wjakethompson/jayhawkdown</a>).</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="1-introduction.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": [["thesis.pdf", "PDF"], ["thesis.docx", "Word"]],
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
