% *============================================================================*
%                  UNIVERSITY OF MELBOURNE THESIS TEMPLATE
% *============================================================================*

% +----------------------------------------------------------------------------+
%  This is a Pandoc/LaTeX template for a University of Melbourne thesis designed
%  to be used as part of a bookdown project
%  (https://bookdown.org/yihui/bookdown/).
% +----------------------------------------------------------------------------+

% +----------------------------------------------------------------------------+
%  Changes:
%
%  2018-10-10 (Luke Zappia)
%    * Create first draft template
% +----------------------------------------------------------------------------+

% *============================================================================*
%  PREAMBLE
%
%  Set up the document, load packages, set parameters etc.
% *============================================================================*

% +-----Document class---------------------------------------------------------+
%  Set the class associated with the document, options in square brackets are
%  passed to the class
% +----------------------------------------------------------------------------+

\documentclass[11pt,a4paper,titlepage,twoside,openright]{style/unimelbthesis}

% +-----Packages---------------------------------------------------------------+
%  External packages used in the document
% +----------------------------------------------------------------------------+

\usepackage{amsmath}   % American Mathematics Society standards
\usepackage{amsxtra}   % Additional math symbols
\usepackage{amssymb}   % Additional math symbols
\usepackage{amsthm}    % Additional math symbols
\usepackage{latexsym}  % Additional math symbols
\usepackage{booktabs}  % Table formatting
\usepackage{longtable} % Table formatting
\usepackage{hyperref}  % Hyperlinks
\usepackage{setspace}  % Line spacing
\usepackage{chemarr}   % Improved reaction arrows for chemists
\usepackage{palatino}  % Use the palatino font family
\usepackage{mathpazo}  % Use the palotino font family
\usepackage{graphicx}  % Extended graphics package

% +-----Parameters-------------------------------------------------------------+
%  Set parameters for the document. To convert from YAML to LaTeX we need to add
%  the dollar signs.
% +----------------------------------------------------------------------------+

\title{Tools and techniques for single-cell RNA-seq data}
\author{Luke Zappia}
\orcid{0000-0001-7744-8565}
\degree{Doctor of Philosophy}
\submissionmonth{November}
\submissionyear{2018}
\department{School of Biosciences}
\university{The University of Melbourne}
\statement{Submitted in Total Fulfillment of the Requirements of the Degree of Doctor of Philosophy}

% +-----Syntax highlighting----------------------------------------------------+
%  If code chunks are included in the document this allows Pandoc to insert the
%  code highlighting macros.
% +----------------------------------------------------------------------------+


% *============================================================================*
%  FRONT MATTER
%
%  Front section of the document, everything before the main text.
% *============================================================================*

\usepackage{amsthm}
\newtheorem{theorem}{Theorem}[chapter]
\newtheorem{lemma}{Lemma}[chapter]
\theoremstyle{definition}
\newtheorem{definition}{Definition}[chapter]
\newtheorem{corollary}{Corollary}[chapter]
\newtheorem{proposition}{Proposition}[chapter]
\theoremstyle{definition}
\newtheorem{example}{Example}[chapter]
\theoremstyle{definition}
\newtheorem{exercise}{Exercise}[chapter]
\theoremstyle{remark}
\newtheorem*{remark}{Remark}
\newtheorem*{solution}{Solution}
\begin{document}

\begin{frontmatter}

% +-----Title------------------------------------------------------------------+

  \maketitle

% +-----Abstract---------------------------------------------------------------+

  \begin{abstract}
    The preface pretty much says it all.
    
    \par
    
    Second paragraph of abstract starts here.
  \end{abstract}

% +-----Declaration------------------------------------------------------------+

  \begin{declaration}
    This is to certify that:
    
    \begin{enumerate}
    \def\labelenumi{\roman{enumi}.}
    \tightlist
    \item
      the thesis comprises only their original work towards the {[}name of the award{]} except where indicated in the preface;
    \item
      due acknowledgement has been made in the text to all other material used; and
    \item
      the thesis is fewer than the maximum word limit in length, exclusive of tables, maps, bibliographies and appendices or that the thesis is {[}number of words{]} as approved by the Research Higher Degrees Committee.
    \end{enumerate}
  \end{declaration}

% +-----Preface----------------------------------------------------------------+

  \begin{preface}
    This is an example of a thesis setup to use the reed thesis document class (for LaTeX) and the R bookdown package, in general.
  \end{preface}

% +-----Acknowledgements-------------------------------------------------------+

  \begin{acknowledgements}
    This template is based on thesisdown (\url{https://github.com/ismayc/thesisdown}) and makes use of RMarkdown (\url{https://rmarkdown.rstudio.com/}) and bookdown \url{https://bookdown.org/yihui/bookdown/}. The LaTeX template is based on John Papandriopoulos' University of Melbourne thesis template (\url{https://github.com/jpap/phd-thesis-template}). Inspriation also comes from similar projects including beaverdown (\url{https://github.com/zkamvar/beaverdown}), aggidown (\url{https://github.com/ryanpeek/aggiedown}), huskydown (\url{https://github.com/benmarwick/huskydown}) and jayhawkdown (\url{https://github.com/wjakethompson/jayhawkdown}).
  \end{acknowledgements}

% +-----Table of contents------------------------------------------------------+

  \hypersetup{linkcolor=black}
  \setcounter{tocdepth}{2}
  \tableofcontents

% +-----List of tables---------------------------------------------------------+

  \listoftables

% +-----List of figures--------------------------------------------------------+

  \listoffigures

% +-----List of copyright------------------------------------------------------+


\end{frontmatter}

% *============================================================================*
%  MAIN MATTER
%
%  Main part of the document. Includes all chapters and appendices.
% *============================================================================*


\begin{mainmatter}

\hypertarget{introduction}{%
\chapter{Introduction}\label{introduction}}

\hypertarget{rna-sequencing}{%
\section{RNA sequencing}\label{rna-sequencing}}

\begin{itemize}
\tightlist
\item
  Central dogma
\item
  Why RNA-seq?
\end{itemize}

\hypertarget{capture-and-reverse-transcription}{%
\subsection{Capture and reverse transcription}\label{capture-and-reverse-transcription}}

\begin{itemize}
\tightlist
\item
  PolyA capture
\item
  Ribosomal depletion
\end{itemize}

\hypertarget{high-throughput-sequencing}{%
\subsection{High-throughput sequencing}\label{high-throughput-sequencing}}

\begin{itemize}
\tightlist
\item
  Illumina sequencing

  \begin{itemize}
  \tightlist
  \item
    Sequence by synthesis
  \end{itemize}
\end{itemize}

\hypertarget{analysis-of-rna-seq-data}{%
\subsection{Analysis of RNA-seq data}\label{analysis-of-rna-seq-data}}

\begin{itemize}
\tightlist
\item
  Experimental design
\item
  Negative binomial
\item
  Normalisation
\item
  Differential expression testing
\end{itemize}

\hypertarget{single-cell-capture-technologies}{%
\section{Single-cell capture technologies}\label{single-cell-capture-technologies}}

\begin{itemize}
\tightlist
\item
  First protocol
\item
  Fluidigm
\end{itemize}

\hypertarget{droplet-based-cell-capture}{%
\subsection{Droplet based cell capture}\label{droplet-based-cell-capture}}

\begin{itemize}
\tightlist
\item
  Drop-seq
\item
  Indrop
\item
  10x Chromium
\end{itemize}

\hypertarget{unique-molecular-identifiers}{%
\subsection{Unique Molecular Identifiers}\label{unique-molecular-identifiers}}

\begin{itemize}
\tightlist
\item
  Why?
\item
  How they work
\end{itemize}

\hypertarget{recent-advances}{%
\subsection{Recent advances}\label{recent-advances}}

\begin{itemize}
\tightlist
\item
  New capture methods
\item
  CITE-seq
\item
  Cell hashing
\end{itemize}

\hypertarget{features-of-single-cell-rna-seq-data}{%
\section{Features of single-cell RNA-seq data}\label{features-of-single-cell-rna-seq-data}}

\begin{itemize}
\tightlist
\item
  Why use single-cell?
\item
  Low counts

  \begin{itemize}
  \tightlist
  \item
    Dropout
  \item
    Bursting
  \item
    Biology
  \end{itemize}
\end{itemize}

\hypertarget{analysing-scrna-seq-data}{%
\section{Analysing scRNA-seq data}\label{analysing-scrna-seq-data}}

\hypertarget{pre-processing-and-quality-control}{%
\subsection{Pre-processing and quality control}\label{pre-processing-and-quality-control}}

\begin{itemize}
\tightlist
\item
  Alignment
\item
  Droplet selection
\item
  UMIs
\item
  Doublet detection
\item
  Bad cells
\item
  Gene filtering
\item
  Cell ranger
\item
  scater
\end{itemize}

\hypertarget{integrating-multiple-datasets}{%
\subsection{Integrating multiple datasets}\label{integrating-multiple-datasets}}

\begin{itemize}
\tightlist
\item
  Why?
\item
  Seurat CCA
\item
  New methods
\end{itemize}

\hypertarget{grouping-cells}{%
\subsection{Grouping cells}\label{grouping-cells}}

\begin{itemize}
\tightlist
\item
  Clustering
\item
  Seurat
\item
  Other approaches
\end{itemize}

\hypertarget{ordering-cells}{%
\subsection{Ordering cells}\label{ordering-cells}}

\begin{itemize}
\tightlist
\item
  Pseudotime
\item
  Monocle
\item
  Other approaches
\end{itemize}

\hypertarget{gene-detection-and-interpretation}{%
\subsection{Gene detection and interpretation}\label{gene-detection-and-interpretation}}

\begin{itemize}
\tightlist
\item
  DE
\item
  Marker genes

  \begin{itemize}
  \tightlist
  \item
    Alternatives - Gini, classifiers
  \end{itemize}
\item
  Reviews
\item
  Classification
\end{itemize}

\hypertarget{kidney-development}{%
\section{Kidney development}\label{kidney-development}}

\hypertarget{structure-and-function}{%
\subsection{Structure and function}\label{structure-and-function}}

\begin{itemize}
\tightlist
\item
  Kidney structure
\item
  Nephron structure
\item
  Important cell types
\end{itemize}

\hypertarget{stages-of-development}{%
\subsection{Stages of development}\label{stages-of-development}}

\begin{itemize}
\tightlist
\item
  Lineage
\item
  Important genes
\end{itemize}

\hypertarget{growing-kidney-organoids}{%
\subsection{Growing kidney organoids}\label{growing-kidney-organoids}}

\begin{itemize}
\tightlist
\item
  Why?

  \begin{itemize}
  \tightlist
  \item
    Disease modelling
  \end{itemize}
\item
  Protocol
\item
  Growth factors
\item
  Characterisation
\item
  Reproducibility
\end{itemize}

\hypertarget{the-scrna-seq-tools-landscape}{%
\chapter{The scRNA-seq tools landscape}\label{the-scrna-seq-tools-landscape}}

\hypertarget{simulating-scrna-seq-data}{%
\chapter{Simulating scRNA-seq data}\label{simulating-scrna-seq-data}}

\hypertarget{introduction-1}{%
\section{Introduction}\label{introduction-1}}

\hypertarget{splatter-publication}{%
\section{Splatter publication}\label{splatter-publication}}

\frame{\includegraphics[page=1, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=2, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=3, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=4, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=5, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=6, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=7, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=8, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=9, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=10, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=11, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=12, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=13, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=14, width=\textwidth]{figures/splatter-paper-cropped}}
\frame{\includegraphics[page=15, width=\textwidth]{figures/splatter-paper-cropped}}

\hypertarget{visualising-clustering-across-resolutions}{%
\chapter{Visualising clustering across resolutions}\label{visualising-clustering-across-resolutions}}

\hypertarget{analysis-of-kidney-organoid-scrna-seq-data}{%
\chapter{Analysis of kidney organoid scRNA-seq data}\label{analysis-of-kidney-organoid-scrna-seq-data}}

\hypertarget{conclusion}{%
\chapter{Conclusion}\label{conclusion}}

\hypertarget{references}{%
\chapter*{References}\label{references}}
\addcontentsline{toc}{chapter}{References}

\hypertarget{refs}{}

\end{mainmatter}

\end{document}
